nuxeo-distribution/nuxeo-launcher/src/main/java/org/nuxeo/launcher/NuxeoLauncher.java

```diff
@@ -1647,13 +1620,26 @@ public boolean registerTrial() throws IOException, ConfigurationException, Packa
             return false;
         }
         log.info(String.format(
-                "Trial registered to %s for project %s\nDescription: %s\n"
+                "Trial registered to %s for project %s%n"
                         + "Please ensure you have validated your registration with the confirmation email before starting the server.",
-                registration.get(REGISTRATION_EMAIL), registration.get(REGISTRATION_PROJECT),
-                registration.get(REGISTRATION_DESCRIPTION)));
+                registration.get(TrialField.EMAIL.getId()), registration.get(TrialField.PROJECT.getId())));
         return true;
     }
 
+    protected void putIfValid(Map<String, String> map, String userInput, TrialField field)
```

Comment:
Not sure if I need to add `@since` for protected methods. @funshodavid said it's not necessary in [this comment](https://github.com/nuxeo/nuxeo/pull/873#discussion_r122388959), while you said I should... I don't have any opinion about it, please let me know the decision. And it would be a plus if this rule could be documented to the contributing guide line.